handbook/brand.md

```diff
@@ -270,6 +270,73 @@ When deciding whether to describe Fleet as open source or open core, define who
 
 > For simplicity and to avoid conflicts with other uses of the word "core" (such as "core product" or "core team"), Fleet is always described as simply **"open source"** in all writing and verbal communication. In specific situations, e.g., discussing the distinction between various kinds of open source, it can be appropriate to mention **"open core"** to clarify your meaning. If ever in doubt, go with "open source."
 
+## Commonly used terms
+
+| Term | Meaning | 
+|:------ |:-----------------|
+| **antivirus** | A class of programs designed to detect, block, and clear away malware from devices, networks, and IT systems. |
+| **API** | (Application Programming Interface) a software go-between that allows applications to communicate.  |
+| **automation** | A system that operates without needing intervention from a human to do so. |
+| **AWS** | (Amazon Web Services) An ever-evolving cloud computing platform designed to allow application providers, ISVs, and vendors to host applications. |
+| **CI/CD** | (Continuous Integration and Continuous Delivery/Continuous Deployment) A software development practice where cumulative code changes are made regularly and accurately. |
+| **CLI** | (Command Line Interface) A tool for managing Fleet from the command line. |
+| **Client Platform Engineer (CPE)** | See: CPE. |
+| **cloud** | Data storage and access through the internet instead of a device's hard drive. |
```

Comment:
```suggestion
| **cloud** | Data storage, networking, servers, databases, software, intelligence, and analytics through the internet instead of a device's hard drive. |
```

Reply:
How is this @GuillaumeRoss?

Reply:
Will commit it in a batch for you!